### PR TITLE
Set package version (Closes #30)

### LINKED ISSUE #30: Set petab.__version__

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,10 +11,12 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+# read version from file
+exec(read(os.path.join('petab', 'version.py')))
```
[the remark this answers is not in the document]
Comment (yannikschaelte) [reply setup.py]: Should work so:
```python
with open(os.path.join(os.path.dirname(__file__),
          "pypesto", "version.py")) as f:
    version = f.read().split('\n')[0].split('=')[-1].strip(' ').strip('"')
```

[the remark this answers is not in the document]
Comment (yannikschaelte) [reply setup.py]: maybe also in ``petab.__init__``:
```python
from .version import __version__  # noqa: F401
```

[the remark this answers is not in the document]
Comment (yannikschaelte) [reply setup.py]: (to avoid pep8 complaints)

[the remark this answers is not in the document]
Comment (dweindl) [reply setup.py]: > Should work so:
> 
> ```python
> with open(os.path.join(os.path.dirname(__file__),
>           "pypesto", "version.py")) as f:
>     version = f.read().split('\n')[0].split('=')[-1].strip(' ').strip('"')
> ```

I would rather switch of or ignore this codacy check. It don't see any securty implications here.

[the remark this answers is not in the document]
Comment (yannikschaelte) [reply setup.py]: yes, here not. but in general codacy may be right that arbitrary executions of external functions can lead to security problems. but sometimes, it's just overly protective, it also gets a hickup every time it sees the word pickle.

**Pull request merged**